src/test/java/com/easypost/beta/EndShipperTest.java

```diff
@@ -0,0 +1,100 @@
+package com.easypost.beta;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+
+import com.easypost.EasyPost;
+import com.easypost.Fixture;
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.beta.EndShipper;
```

Comment:
One quick Q: Is this standard in Java to place modules inside this `model` directory? If so, can you provide some documentation on the subject here for posterity?

Reply:
It's not a standard in Java. I created the `beta` folder in the `model` directory because that's where all the classes are hosted, also being consistent with other libs Python and Ruby where we put the `beta` folder in the same directory.